app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CardsMailer < ActionMailer::Base
+  default from: 'info@stavcdflashcard.com'
+
+  def pending_cards_notification(user, review_count)
+    @user = user
+    @review_count = review_count
+    @url = 'http://stavcdflahcards.herokuapp.com'
+    mail(to: @user.email, subject: 'Пришло время проверить знания')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.